neuralmonkey/server/server.py

```diff
@@ -15,7 +16,44 @@
 APP.config["args"] = None
 
 
+def root_dir():  # pragma: no cover
+    return os.path.abspath(os.path.dirname(__file__))
+
+
+def get_file(filename):  # pragma: no cover
+    src = os.path.join(root_dir(), filename)
+    return open(src).read()
+
+
+def translate(data):  # pragma: no cover
```

Comment:
Já bych tomu neříkal translate, to je prostě run, pouští to runnery bez ohledu na to, co jsou zač.

Reply:
translate() zmeneno na run()